config/shuttle.go

```diff
@@ -108,8 +109,39 @@ func NewShuttle() *Shuttle {
 				TargetMessageSize:          16 << 10,
 			},
 			NoLimiter: true,
+			LimitsConfig: Limits{
+				SystemLimitConfig: SystemLimit{
+					MinMemory:      1 << 30,
+					MaxMemory:      10 << 30,
+					MemoryFraction: .2,
+
+					StreamsInbound:  64 << 10,
+					StreamsOutbound: 128 << 10,
+					Streams:         256 << 10,
+
+					ConnsInbound:  256,
+					ConnsOutbound: 256,
+					Conns:         1024,
+
+					FD: 8192,
+				},
+				TransientLimitConfig: TransientLimit{
+					StreamsInbound:  2 << 10,
+					StreamsOutbound: 4 << 10,
+					Streams:         4 << 10,
+
+					ConnsInbound:  256,
+					ConnsOutbound: 256,
+					Conns:         512,
+
+					FD: 1024,
+				},
+			},
+		},
+		ConnectionManagerConfig: ConnectionManager{
+			LowWater:  2000,
+			HighWater: 3000,
```

Comment:
I'm a bit unsure as to where all these values came from. It seems you got them to the previous code, but it'd be good to have @whyrusleeping help us put some useful comments here.